openstates/nv/bills.py

```diff
@@ -245,8 +245,18 @@ def scrape_actions(self, root, bill, actor):
                         action_type = atype
                         break
 
+
+                if "Committee on" in action:
+                    committees = re.findall("Committee on ([a-zA-Z, ]*)\.",action)
```

Comment:
What if there's text _after_ the committee name? How do you know when to stop the `re` search?